packages/generator-single-spa/src/generator-single-spa.js

```diff
@@ -128,7 +129,7 @@ async function runFrameworkGenerator() {
         type: "list",
         name: "framework",
         message: "Which framework do you want to use?",
-        choices: ["react", "vue", "angular", "svelte", "other"],
+        choices: ["react", "vue", "angular", "svelte", "html", "other"],
```

Comment:
I debated calling this `"html/web-component"` but the generated repo doesn't have any web-components code. Should it be left as-is and we can create another generator for web-components? Or should the example app lean into some web-component lib and use that?

Reply:
Single-spa-html is not popular at all, from what I've seen. I don't know of anyone actually using it. So it's hard to say what people would expect the generator to create. I think people get excited about the idea of web components and would want some kind of boilerplate. Perhaps a basic lit-html template to help them get started would help, idk.